data-otservbr-global/scripts/creaturescripts/others/player_death.lua

```diff
@@ -70,14 +70,8 @@ function playerDeath.onDeath(player, corpse, killer, mostDamageKiller, unjustifi
 	-- Start Webhook Player Death
 	local playerName = player:getName()
 	local playerLevel = player:getLevel()
-	local killerLink = string.gsub(killerName, "%s+", "+")
-	local playerLink = string.gsub(playerName, "%s+", "+")
-	local serverURL = getConfigInfo("url")
-	if killer and killer:isPlayer() then
-		Webhook.sendMessage(playerName .. " just got killed!", "**[" .. playerName .. "](" .. serverURL .. "/?characters/" .. playerLink .. ")** got killed at level " .. playerLevel .. " by **[" .. killerName .. "](" .. serverURL .. "/?characters/" .. killerLink .. ")**", WEBHOOK_COLOR_OFFLINE, announcementChannels["player-kills"])
-	else
-		Webhook.sendMessage(playerName .. " has just died!", "**[" .. playerName .. "](" .. serverURL .. "/?characters/" .. playerLink .. ")** died at level " .. playerLevel .. " by " .. killerName, WEBHOOK_COLOR_WARNING, announcementChannels["player-kills"])
-	end
+	local vocation = player:vocationAbbrev()
```

Comment:
we can remove this, because isn't in use. We are using in `data/events/scripts/player.lua`
```suggestion
```